bookshelf/src/main/java/com/example/managedvms/gettingstartedjava/auth/Oauth2CallbackServlet.java

```diff
@@ -56,15 +62,25 @@
   @Override
   public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException,
       ServletException {
+    Map<String, Cookie> cookieMap = new HashMap<>();
+    Cookie[] cookies = req.getCookies();
+    for (Cookie c : cookies) {
```

Comment:
Do we always want to do this?   Shouldn't there be a comment to disable this in production?